### ISSUE TITLE
Problem on run adka

### ISSUE THREAD

**Issue by igotfr** [apiel/adka#1]
$ deno --version
deno 1.2.0
v8 8.5.216
typescript 3.9.2

$ deno run --unstable --allow-read --allow-write --allow-env --allow-net https://raw.githubusercontent.com/apiel/adka/master/adka.ts

Check https://raw.githubusercontent.com/apiel/adka/master/adka.ts
error: TS2345 [ERROR]: Argument of type 'string | URL' is not assignable to parameter of type 'string'.
  Type 'URL' is not assignable to type 'string'.
  return new URL(url).pathname
                 ~~~
    at https://deno.land/std@0.60.0/path/win32.ts:917:18

TS2345 [ERROR]: Argument of type 'string | URL' is not assignable to parameter of type 'string'.
  Type 'URL' is not assignable to type 'string'.
  return new URL(url).pathname;
                 ~~~
    at https://deno.land/std@0.60.0/path/posix.ts:438:18

Found 2 errors.

**Comment by apiel** [comment]
Unfortunatelly this is an issue with deno 1.2.0 https://github.com/denoland/deno/issues/5365 
The problem with deno is that they introduce breaking change on each release. Also they are missing unit test so the introduce regression...
I don't really know if I can solve it, unless I change the thirdparty dependency. Till this got fixed, I would recommand to use v1.1.1.

**Comment by apiel** [comment]
I updated adka_server with the latest version of oak and seem to have solve the problem. Try to re-install adka.
